include/ck/tensor_operation/gpu/device/impl/device_grouped_conv_utils.hpp

```diff
@@ -9,9 +9,11 @@ namespace ck {
 namespace tensor_operation {
 namespace device {
 
-template <index_t NumDTensor>
+template <index_t NumATensor = 1, index_t NumBTensor = 1, index_t NumDTensor = 0>
 struct ComputePtrOffsetOfStridedBatch
```

Comment:
I wonder if things won't be more clear if you would create two (SFINAE) specializations of this struct based on the number of A,B tensors? Then you could have clear API and implementation just for specific case, without a need for all `else` branches with just `static_assert`.